cloudbridge/cloud/providers/gce/provider.py

```diff
@@ -5,11 +5,13 @@
 
 
 from cloudbridge.cloud.base import BaseCloudProvider
+import httplib2
```

Comment:
Thanks for pointing this out. The discussion in that thread was very interesting. For this particular GCEInstanceService implementation, we actually create a new Http() object for each request, as suggested here https://developers.google.com/api-client-library/python/guide/thread_safety